dht_net.go

```diff
@@ -63,6 +64,7 @@ func (dht *IpfsDHT) handleNewMessage(s inet.Stream) {
 		}
 
 		// send out response msg
+		s.SetWriteDeadline(time.Now().Add(dhtMessageTimeout))
```

Comment:
Probably best to fix https://github.com/whyrusleeping/go-multiplex/issues/7 first.